test/system/060-mount.bats

```diff
@@ -173,6 +173,48 @@ load helpers
     run_podman rm -t 0 -f $cid
 }
 
+@test "podman mount containers.conf" {
+    dest=/$(random_string 30)
+    tmpfile1=$PODMAN_TMPDIR/volume-test1
+    random1=$(random_string 30)
+    echo $random1 > $tmpfile1
+
+    tmpfile2=$PODMAN_TMPDIR/volume-test2
+    random2=$(random_string 30)
+    echo $random2 > $tmpfile2
+    bogus=$(random_string 10)
+
+    mountStr1=type=bind,src=$tmpfile1,destination=$dest,ro
+    mountStr2=type=bind,src=$tmpfile2,destination=$dest,ro
+    containersconf=$PODMAN_TMPDIR/containers.conf
+    cat >$containersconf <<EOF
+[containers]
+mounts=[ "$mountStr1", ]
+EOF
+    badcontainersconf=$PODMAN_TMPDIR/badcontainers.conf
+    cat >$badcontainersconf <<EOF
+[containers]
+mounts=[ "type=$bogus,src=$tmpfile2,destination=$dest,ro", ]
+EOF
+
+    run_podman 1 run $IMAGE cat $dest
+    is "$output" "cat: can't open '$dest': No such file or directory" "$dest does not exist"
+
+    CONTAINERS_CONF_OVERRIDE="$containersconf" run_podman run $IMAGE cat $dest
+    is "$output" "$random1" "file should contain $random1"
+
+    CONTAINERS_CONF_OVERRIDE="$containersconf" run_podman run --mount $mountStr2 $IMAGE cat $dest
+    is "$output" "$random2" "overridden file should contain $random2"
+
+    CONTAINERS_CONF_OVERRIDE="$containersconf" run_podman 125 run --mount $mountStr1 --mount $mountStr2 $IMAGE cat $dest
+    is "$output" "Error: $dest: duplicate mount destination" "Should through duplicate destination error for $dest"
```

Comment:
typo, should be 'throw'.

And, a broad note about test names/descriptions: it isn't usually necessary to include the expected-string as part of the error name. That gives an error like:
```
   FAIL: Should through duplicate destination error for /sdfsdfdsf
     expected: Error: /sdfsdfsdf: duplicate mount destination
     actual:   something else
```
My preference is to make the name/title something that describes the condition being tested, so in this case something like "podman run with conflicting --mount".

This applies to all the checks above and below.